### PR TITLE
local storage and code cleanup

### PR DESCRIPTION
**Pull request by xavluiz**

### CODE REVIEW

File: src/Util.ts [hunk 1]
```diff
@@ -250,15 +204,18 @@ export function getLocalREADMEFile() {
...
 }
 
 export function getLogId() {
   return 'MusicTime';
 }
 
-export function logIt(message: string) {
+export function logIt(message: string, isError: boolean = false) {
   outputChannel.appendLine(`${formatISO(new Date())} ${getLogId()}: ${message}`);
+	if (isError) {
```
Comment (bjacobson26) [inline src/Util.ts:216]: weird indentation here or is this GitHub showing it weird?

File: src/extension.ts [hunk 2]
```diff
@@ -11,15 +11,18 @@ import { initializeSpotify } from "./managers/PlaylistDataManager";
...
 let currentColorKind: number = undefined;
+let storageManager: LocalStorageManager | undefined = undefined;
 
 export function deactivate(ctx: ExtensionContext) {
   clearWebsocketClient();
   clearSpotifyAccessToken();
 }
 
 export async function activate(ctx: ExtensionContext) {
+	storageManager = LocalStorageManager.getInstance(ctx);
```
Comment (bjacobson26) [inline src/extension.ts:25]: indentation inconsistent?

File: src/managers/DeviceManager.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import { execCmd } from "./ExecManager";
+
+
```
Comment (bjacobson26) [inline src/managers/DeviceManager.ts:3]: extra line

**Review by bjacobson26 — APPROVED** [review]
might be good to add eslint to catch some things. good cleanup 👍

File: src/extension.ts (hunk 2 above)
Comment (xavluiz) [reply src/extension.ts:26]: yeah, updated it with eslint and ran fix-it

File: src/Util.ts (hunk 1 above)
Comment (xavluiz) [reply src/Util.ts:216]: yeah, i think its github but I've updated the package with eslint and cleaned up the files

**Pull request merged**